resources/js/src/app/store/index.js

```diff
@@ -16,6 +16,16 @@ import eventPropagation from "store/plugins/EventPropagationPlugin";
 Vue.use(require("vue-script2"));
 Vue.options.delimiters = ["${", "}"];
 
+Vue.mixin({
+    created()
+    {
+        if (this.$props && this.$props.template)
```

Comment:
https://github.com/plentymarkets/plugin-ceres/pull/1000/files

Merge dir vllt mal den branch rein und binde das in die Struktur der mixins ein.